src/tool_getparam.c

```diff
@@ -2996,11 +2996,7 @@ ParameterError parse_args(struct GlobalConfig *global, int argc,
   struct OperationConfig *config = global->first;
 
   for(i = 1, stillflags = TRUE; i < argc && !result; i++) {
-#ifdef UNDER_CE
-    orig_opt = strdup(argv[i]);
-#else
     orig_opt = curlx_convert_tchar_to_UTF8(argv[i]);
```

Comment:
```suggestion
#ifdef UNDER_CE
    orig_opt = argv[i];
#else
    orig_opt = curlx_convert_tchar_to_UTF8(argv[i]);
#endif
```
Windows CE is weird: It's Unicode, but the `main()` args are non-Unicode.

Reply:
Then we should fix the ifdef in curl_multibyte.h I think

Reply:
in fact, I think I'll make a copy of this for the tool and leave the libcurl version as-is